pom.xml

```diff
@@ -89,11 +89,16 @@
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.249.x</artifactId>
+        <artifactId>bom-2.263.x</artifactId>
         <version>24</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>io.jenkins.plugins</groupId>
+        <artifactId>snakeyaml-api</artifactId>
+        <version>1.29.1</version>
+      </dependency>
```

Comment:
contained in bom version `918.vae501d2cdc99`  - is there a reason we are not updating the bom to pickup the newer versions?